mtags/src/main/scala-3/scala/meta/internal/pc/NamesInScope.scala

```diff
@@ -12,27 +13,29 @@ import dotty.tools.dotc.core.Names._
 import dotty.tools.dotc.core.StdNames._
 import dotty.tools.dotc.core.Symbols._
 import dotty.tools.dotc.core.Types._
+import dotty.tools.dotc.typer.ImportInfo
 
 case class NamesInScope(
-    values: Map[String, Symbol]
+    values: Map[String, Value]
```

Comment:
This needs to be reworked a bit when it comes to renames, because of the conflict with my PR. Sorry about that! Shouldn't be too crazy though hopefully.

Reply:
Don't worry! I've seen these changes and planned to rework this part.

Reply:
Thanks!